age/src/scrypt.rs

```diff
@@ -38,8 +38,24 @@ fn target_scrypt_work_factor() -> u8 {
             SystemTime::now().duration_since(start).ok()
         }
 
+        // Platforms that can use Performance timer
+        #[cfg(all(target_arch = "wasm32", feature = "web-sys"))]
```

Comment:
This looks like it will conflict with the `SystemTime` variant if `web-sys` were enabled for `target_os = "wasi"`. Let's change this to:
```suggestion
        #[cfg(all(
            target_arch = "wasm32",
            not(target_os = "wasi"),
            feature = "web-sys"
        ))]
```